.codecov.yml

```diff
@@ -0,0 +1,12 @@
+---
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 1
+        base: auto
+comment: false
```

Comment:
You should also be able to make the comment smaller with something like:

```
comment:
  layout: "diff"
```

Either `diff` or `files` is probably the most useful part.

See: https://docs.codecov.io/docs/pull-request-comments